src/utils/defaultStyle.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
```

Comment:
this is a wrapper around the new hooks based substyle API, which we currently cannot use directly since hooks are not supported for class-based components